logger.js

```diff
@@ -138,16 +137,20 @@ function pinoLogger (opts, stream) {
 
     req.id = genReqId(req, res)
 
-    const log = quietReqLogger ? logger.child({ [requestIdKey]: req.id }) : logger
+    const reqLog = quietReqLogger ? logger.child({ [requestIdKey]: req.id }) : logger
+    let fullReqLogger = reqLog.child({ [reqKey]: req })
+
+    const resLog = quietResLogger ? logger.child({ [requestIdKey]: req.id }) : logger
+    let fullResLogger = resLog.child({ [resKey]: res })
```

Comment:
> What errors are you getting?

@mcollina a bunch of tests stop passing after adding this line

Reply:
The tests should be updated

Reply:
could we not do this?

```
const log = quietReqLogger  || quietResLogger ? logger.child({ [requestIdKey]: req.id }) : logger
```

and leave the rest of the code as is?

Reply:
hm, I just did it but the tests are still not passing 

![image](https://user-images.githubusercontent.com/13461315/203825473-046cee48-6c95-4560-9dc5-ee8bdf412f29.png)

Reply:
I actually really want this feature, I'll pull tomorrow and use some work time on it